src/config.c

```diff
@@ -524,41 +677,93 @@ void ConfigInit(RedisModuleCtx *ctx,
                                                                                                      REDISMODULE_CONFIG_HIDDEN,                  getString,  setString,  NULL, c);
 
                                                    /* name */                  /* default-value */   /* flags */
-    ret |= RedisModule_RegisterEnumConfig(ctx,    CONF_LOGLEVEL,                   LOG_LEVEL_NOTICE, REDISMODULE_CONFIG_DEFAULT,  redisraft_loglevels, redisraft_loglevel_enums, LOG_LEVEL_COUNT, getEnum, setEnum, NULL, c);
-    ret |= RedisModule_RegisterEnumConfig(ctx,    CONF_TRACE,                      TRACE_OFF,        REDISMODULE_CONFIG_BITFLAGS, trace_names,         trace_flags,              TRACE_COUNT,     getEnum, setEnum, NULL, c);
+    ret |= RedisModule_RegisterEnumConfig(ctx,    CONF_LOGLEVEL,                   LOG_LEVEL_NOTICE, REDISMODULE_CONFIG_DEFAULT,  redisraft_loglevels, redisraft_loglevel_enums, LOG_LEVEL_COUNT,  getEnum, setEnum, NULL, c);
```

Comment:
```suggestion
    ret |= RedisModule_RegisterEnumConfig(ctx,    CONF_LOGLEVEL,                   LOG_LEVEL_NOTICE, REDISMODULE_CONFIG_DEFAULT,  redisraft_loglevels, redisraft_loglevel_enums, LOG_LEVEL_COUNT, getEnum, setEnum, NULL, c);
```
remove space

Reply:
There is actually no misalignment here. Open the file itself, it should look okay. (These lines are too long but I don't see a better alternative for readability).